CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -963,6 +975,7 @@ local function CreateSettingsHub()
 		
 		this.ClippingShield.Parent = panel:GetGUI()
 		this.Shield.Parent.ClipsDescendants = false
+		this.VRShield.Visible = true
```

Comment:
Should we also handle the case of turning off VR for future proofing? That could be in a separate commit though.